backend/src/resolvers/categories/category.ts

```diff
@@ -0,0 +1,49 @@
+import { Category } from "../../entities/Category";
+import { Arg, Mutation, Resolver } from "type-graphql";
+import { CategoryResponse } from "../response";
+import { CategoryInput } from "../categoryInput";
+import { validateCategory } from "../../utils/validateCategory";
+import { getConnection } from "typeorm";
+
+@Resolver(Category)
+export default class CategoryResolver {
+  @Mutation(() => CategoryResponse)
+  async createCategory(@Arg("inputs") inputs: CategoryInput): Promise<CategoryResponse> {
```

Comment:
add is auth middleware here to check if user is connected or not
@UseMiddleware(isAuth)